app/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,9 @@
   <string name="fraction_error_invalid_format">Please enter a valid fraction (e.g., 5/3 or 1 2/3)</string>
   <string name="fraction_error_divide_by_zero">Please do not put 0 in the denominator</string>
   <string name="fraction_error_larger_than_seven_digits">None of the numbers of the fraction should be larger than 7 digits.</string>
+  <string name="number_error_starting_with_floating_point">Please begin your answer with a number(e.g.,”0” in 0.5).</string>
+  <string name="number_error_invalid_format">Please enter a valid number.</string>
+  <string name="number_error_larger_than_seven_digits">Maximum 15 characters.</string>
```

Comment:
Is this the final wording @mschanteltc suggested? It feels a bit context-less. Should we say something like: "The answer can contain at most 15 numbers." or something similar? Also, do users know what 'character' means in this context?

Reply:
@BenHenning In my opinion digit would be a familiar term to kids i.e. "The answer can contain at most 15 digits."

Reply:
If we allow at most 15 "numbers," would this mean the decimal point would be excluded from the count? I agree "characters" can sound a bit out of context. "Characters" was used to describe all text input.

Reply:
Is this ok? "The answer can contain at most 15 characters(i.e.,[0-9] or ".")"

Reply:
> If we allow at most 15 "numbers," would this mean the decimal point (and possibly forward slashes, negative signs, etc.) would be excluded from the count? I agree "characters" can sound a bit out of context. "Characters" was used to describe all text input.
@mschanteltc 
only "."and "-" are allowed, no other characters are allowed in number input.

Reply:
Will other answer inputs (such as words and equations) follow this 15 character max as well? If so, I think we should stick with the word "character" out of consistency.

Reply:
@seanlip PTAL on this and suggest the error message for number too long, currently we are showing "The answer can contain at most 15 characters (i.e.,[0-9] or "."or "-")."

Reply:
@mschanteltc Sorry, missed your question. Other answer inputs will not have this maximum limit.

Perhaps: "The answer can contain at most 15 digits (0-9) or symbols (. or -)."